### PR TITLE
various fixes

### PR DESCRIPTION
**Pull request by iyegoroff**
- updated `ImageProperties` type;
- removed `ScrollViewStyle` type, `ScrollView` uses just `ViewStyle`;
- added `ShadowOffset` type.

### CODE REVIEW

File: src/Fable.Helpers.ReactNative.fs [hunk 1]
```diff
@@ -13,6 +13,11 @@ type Ref<'t> = ('t -> unit)
 
 module Props =
 
+    [<Pojo>]
```
Comment (alfonsogarciacaro) [inline src/Fable.Helpers.ReactNative.fs]: We're thinking of deprecating `Pojo` in Fable 2 as it seems it's not actually necessary with latest React. Will `ShadowOffset` still work if you remove it?

Comment (iyegoroff) [reply src/Fable.Helpers.ReactNative.fs]: Yes, it works without `Pojo`.

Comment (alfonsogarciacaro) [reply src/Fable.Helpers.ReactNative.fs]: Great, thanks for confirming!

**Comment by alfonsogarciacaro** [comment]
Thanks a lot for your efforts! Published as 2.0.0-alpha-004. Just let me know when you think breaking changes are not needed any more and we can release a stable 2.0 version :)

**Pull request merged**